simavr/sim/sim_gdb.c

```diff
@@ -301,9 +301,10 @@ gdb_handle_command(
 				 */
 				gdb_send_reply(g, "1");
 				break;
-			} else if (strncmp(cmd, "Offsets", 7) == 0) {
-				gdb_send_reply(g, "Text=0;Data=800000;Bss=800000");
-				break;
+			// Rmoving the following 3 lines fixes #150 issue:
```

Comment:
In my experience, the problem exists exactly as described in the issue #150, and removing these lines resolves it. I tested it in several Linux flavors with avr-gdb v7.8.2 and v7.12. If anybody have any thoughts, please leave a comment!